docs/references/astro/migrating-from-astro-community-sdk.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: 'Migrating from the Astro community SDK'
+description: Learn how to migrate from the Astro community SDK to the Clerk Astro SDK.
+---
+
+# Migrating from the Astro community SDK
+
+In July 2024, we introduced official support for Astro. This migration guide covers converting from the [`astro-clerk-auth`](https://github.com/panteliselef/astro-with-clerk-auth) community SDK to the official SDK. It covers the breaking changes that were introduced and provides examples on how to resolve them.
+
+
+**Overview:**
+- Install `@clerk/astro`, uninstall `astro-clerk-auth`
+- Update import paths
+- Replace deprecated APIs
+
+## Installation
+
+Uninstall the community SDK and install the new offical SDK for Astro.
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash filename="terminal"
+    npm uninstall astro-clerk-auth
+    npm install @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    yarn remove astro-clerk-auth
+    yarn add @clerk/astro
+    ```
+
+    ```bash filename="terminal"
+    pnpm remove astro-clerk-auth
+    pnpm add @clerk/astro
+    ```
+</CodeBlockTabs>
+
+## Breaking Changes
+
+### The integration
+
+When using the integration from `@clerk/astro` we are now hotloading clerk-js by default.
+Previously you could choose between two strategies when using the clerk intergration. Now only one is offered:
+
+**Hotload** 
+
+```diff
+import { defineConfig } from "astro/config";
+import node from "@astrojs/node";
+- import clerk from "astro-clerk-auth/hotload";
++ import clerk from "@clerk/astro";
+
+- import clerk from "astro-clerk-auth";
++ import clerk from "@clerk/astro";
+
+- import clerk from "astro-clerk-auth/integration";
++ import clerk from "@clerk/astro";
+
+- import clerk from "astro-clerk-auth/integration/hotload";
++ import clerk from "@clerk/astro";
+
+export default defineConfig({
+    integrations: [clerk()],
+    output: "server",
+    adapter: node({ mode: "standalone" }),
+});
+```
```

Comment:
```suggestion
```diff
import { defineConfig } from "astro/config";
import node from "@astrojs/node";
- import clerk from "astro-clerk-auth/hotload";
- import clerk from "astro-clerk-auth";
- import clerk from "astro-clerk-auth/integration";
- import clerk from "astro-clerk-auth/integration/hotload";
+ import clerk from "@clerk/astro";

export default defineConfig({
    integrations: [clerk()],
    output: "server",
    adapter: node({ mode: "standalone" }),
});
```

This looks like 4, not 2, particularly the way its written.
```

I think this makes more sense, as it just have the new import once. This could be a personal feeling though.